### DISCUSSION TITLE
Übergabe bei Genesenen- und Auffrischungsimpfung

### DISCUSSION THREAD

**Discussion by fue-medatixx** [Digitaler-Impfnachweis/certification-apis#35] [Q&A] [answered]
Vorweg: Ein Nachweis für Genesene ist noch in Vorbereitung?! Gibt es hierzu einen zeitlichen Horizont?

Bei der Angabe der Impfdosen übergibt man für z.B. Biontech bei der Erstimpfung 1/2 und bei der Abschlussimpfung 2/2.
Was ist bei der Impfung eines Genesenen zu übertragen? 2/2 oder 1/1? Dieser benötigt ja nur eine Impfung.
Und bis zur Auffrischungsimpfung dauert es auch nicht mehr lange. Wäre das dann 3/3 oder auch 2/2 (getreu dem Motto: 2/2 zeigt immer den vollständigen Impfschutz an)?

Und rein formal: Die Erstimpfung bringt dem Bürger ja nur einen bedingten Mehrwert. Muss / sollte diese dokumentiert werden? Ich gehe davon aus, dass eine Dokumentation der Abschlussimpfung für den dig. Impfnachweis ausreichend ist.

**Comment by mge-medisoftware** [comment, marked as answer]
Es gibt ein separates Datenschema für ein Genesenenzertifikat, siehe "Vertrag über die Erstellung eines Zertifikatmoduls für Primärsysteme zur Ausstellung von COVID-19 Zertifikaten" Anlage 1.2.1 - Annex 2 (vom BMG)

**Reply by oliver-steinbrecher** [reply]
Die [OpenAPI](https://github.com/Digitaler-Impfnachweis/certification-apis/blob/master/dgc-certify-api.yaml) enthält auch die Erzeugung der Genesenenzertifikate. Es ist noch in Klärung, wann dieser Stand auch auf dem öffentlichen System bereitgestellt werden kann.
